tools/vagrant/README.md

```diff
@@ -5,34 +5,35 @@ The following instructions were tested on Mac OS X El Capitan, Ubuntu 14.04.3 LT
 *Requirements*
 - Install [Vagrant](https://www.vagrantup.com/downloads.html)
  
-### Clone the repository and change directory to `tools/vagrant/custom`
+### Clone the repository and change directory to `tools/vagrant`
 ```
-git clone https://github.com/openwhisk/openwhisk.git
-cd openwhisk/tools/vagrant/custom
+git clone --depth=1 https://github.com/openwhisk/openwhisk.git
+cd openwhisk/tools/vagrant
 ```
 
-### Create VM using custom datastore (Cloudant or CouchDB)
-Follow instructions [tools/db/README.md](../../db/README.md) on how to configure a datastore for OpenWhisk.
+### Create a Vagrant VM
 
-#### Option 1: Create VM using Cloudant DB
+#### Option 1: Create VM using ephemeral CouchDB
+**Important** We advise that you use this method for development of OpenWhisk.
+Please note that no data will persist between two creations of the container.
+For more information on datastore configurations see [tools/db/README.md](../../db/README.md).
 ```
-# Provide credentials for cloudant database with admin permissions
-OW_DB=cloudant OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy vagrant up
+# Configure with couchdb docker container running inside the VM
+./hello
 ```
 
-#### Option 2: Create VM using persistent CouchDB
+Follow instructions [tools/db/README.md](../../db/README.md) on how to configure a datastore for OpenWhisk.
+
+#### Option 2: Create VM using Cloudant DB
 ```
-# Provide credentials for couchdb database with admin permissions
-OW_DB=couchdb OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy OW_DB_PROTOCOL=http OW_DB_HOST=1.2.3.4 OW_DB_PORT=5984 vagrant up
+# Provide credentials for cloudant database with admin permissions
+OW_DB=cloudant OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy ./hello
 ```
 
-#### Option 3: Create VM using ephemeral CouchDB
-**Important** We advise that you use this method only as a temporary measure.
-Please note that no data will persist between two creations of the container.
-For more information on datastore configurations see [tools/db/README.md](../../db/README.md).
+#### Option 3: Create VM using persistent CouchDB
 ```
-# Configure with couchdb docker container running inside the VM
-vagrant up
+# Provide credentials for couchdb database with admin permissions
+OW_DB=couchdb OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy OW_DB_PROTOCOL=http OW_DB_HOST=1.2.3.4 OW_DB_PORT=5984 ./hello
```

Comment:
CouchDB? I have to look at where thes are used but from the backend case matters.